website/pages/docs/platform/k8s/helm/index.mdx

```diff
@@ -54,8 +54,8 @@ $ git clone https://github.com/hashicorp/vault-helm.git
 $ cd vault-helm
 
 # Checkout a tagged version
-$ git checkout v0.3.3
+$ git checkout v0.4.0
```

Comment:
Are you staging this for 0.4.0 release?

Reply:
Yeah, I guess I was assuming vault-helm 0.4.0 would be the first release with the helm 3 changes.